docs/user/welcome/faq.rst

```diff
@@ -105,17 +105,6 @@ Q. How are GeoTools versions organized?
 Like many open source projects, GeoTools has a *development* version and one or more *stable* versions active at any
 given time. By active, we mean that the project developers are working on new features, improvements and bug fixes.
 
-Source code for the stable branch(es) can be found in the *branches* folder of the GeoTools subversion repository. For
-example, at the time of writing the active stable branch is `branches/2.7.x <http://svn.osgeo.org/geotools/branches/2.7.x/>`_.
-
-Formal releases are based on a stable branch. The source code for a formal release is archived in the *tags* folder of
-the subversion repository. For example, the version 2.7.1 sources can be found in
-`tags/2.7.1 <http://svn.osgeo.org/geotools/tags/2.7.1/>`_.
-
-Source code for the development version are in the `trunk <http://svn.osgeo.org/geotools/trunk/>`_ folder. This is the
-bleeding edge code where the latest features are being worked on. Eventually this code will become the next stable
-branch.
-
```

Comment:
While geoserver no longer uses subversion, it does use GitHub. Rather than removing this section, update it to match the GeoTools GitHub structure. 
Stable branches are a branch in the GitHub repository. The current stable branch is 14.x (https://github.com/geotools/geotools/tree/14.x).
Releases are a tag in the GitHub repository. The current stable release is 14.0 (https://github.com/geotools/geotools/releases/tag/14.0)
The development branch is master (https://github.com/geotools/geotools)